packages/aerodrome/src/Aerodrome.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  type TransactionFilter,
+  type SwapActionParams,
+  compressJson,
+} from '@rabbitholegg/questdk'
+import { type Address, zeroAddress } from 'viem'
+import { Chains, CHAIN_TO_TOKENS } from '@rabbitholegg/questdk-plugin-utils'
+import {
+  AERODROME_ROUTER,
+  ETH_FOR_TOKENS_FRAGMENTS,
+  TOKENS_FOR_ETH_FRAGMENTS,
+  TOKENS_FOR_TOKENS_FRAGMENTS,
+  WETH_ADDRESS,
+} from './constants'
+
+export const swap = async (
+  _params: SwapActionParams,
+): Promise<TransactionFilter> => {
+  const { chainId, tokenIn, tokenOut, amountIn, amountOut, recipient } = _params
+
+  return compressJson({
+    chainId,
+    value: tokenIn === zeroAddress ? amountIn : undefined,
+    to: AERODROME_ROUTER,
+    input: {
+      $or: [
+        {
+          // swapExactETHForTokens
+          $abi: ETH_FOR_TOKENS_FRAGMENTS,
+          amountOutMin: amountOut,
+          routes: {
```

Comment:
Do we have a helper somewhere for this?

Reply:
We have a helper for this, but we have not put it into our utils package. It is used quite often so maybe it would not be a bad idea to add this helper.

I mostly did it this way because I was making a tutorial during the development of this plugin and wanted to show an example of using some of the filter operators. 

I went ahead and switched it to use the helper function.